packages/docs/src/styleguide.md

```diff
@@ -1,16 +1,18 @@
-Blueprint is a React UI toolkit for the web.
+Blueprint is a React-based UI toolkit for the web.
```

Comment:
I think it's way cleaner without the verboseness of the "-based" (not really necessary). If you feel strongly, I'm cool with here, but I prefer keeping the tagline everywhere as is for now. Note that changing it in this file only isn't enough, there are other places where it appears on GitHub, the landing, different services... Let's wait for 2.0 please, and we can work out and even better one.